cozepy/conversations/message/feedback/__init__.py

```
@@ -0,0 +1,111 @@
+    UNLIKE = "unlike"  # 踩
+
+
+class CreateConversationMessageFeedbackResp(CozeModel):
+    pass
+
+
+class DeleteConversationMessageFeedbackResp(CozeModel):
+    pass
+
+
+class ConversationsMessagesFeedbackClient(object):
```

Pylint (3.3.7)

[refactor] Class 'ConversationsMessagesFeedbackClient' inherits from object, can be safely removed from bases in python3

(R0205)